week-2/Homework/mandatory/3-playing-computer.js

```diff
@@ -6,37 +6,44 @@
 
   Answer the following questions:
 
-  1. This program throws an error. Why? (If you can't find it, try executing it).
+  task 1 
+  1. This program throws an error. Why? (If you can't find it, try executing it).  //because "b" is not defined because it had no varruble with the value./
   2. Remove the line that throws the error.
-  3. What is printed to the console?
-  4. How many times is "f1" called?
-  5. How many times is "f2" called?
-  6. What value does the "a" parameter take in the first "f1" call?
+task 2
+  3. What is printed to the console?  // 2, 6, 4, 9, 6, 13, 8   
+  console.log(x);   2
+  console.log(a);   6
+  console.log(d);   4, 6, 8 
+  console.log(e);   9, 13
+  4. How many times is "f1" called?   // 1st call of f1 is when consol.log(e) = 9,  //  last time f1 is called and resulting in console.log(e) = 13  
+  5. How many times is "f2" called?  // 1st call of f2 is when console.log(d) = 4, second time f2 is called and resulting in console.log(d) = 6   // last time f2 is called and resulting in console.log(d) = 8
+  6. What value does the "a" parameter take in the first "f1" call?        // value 8 
```

Comment:
This is actually the answer to question 7 :) This is asking what value the `a` parameter on line 21 will have, not what the `a` variable on line 19 will have. Can you try this one again?